frontend/js/src/explore/lb-radio/Playlist.tsx

```diff
@@ -0,0 +1,135 @@
+      <div className="feedback-header">Query feedback:</div>
+      <ul>
+        {feedback.map((item: string) => {
+          return <li key={`${item}`}>{`${item}`}</li>;
+        })}
+      </ul>
+    </div>
+  );
+}
+
+export function Playlist(props: PlaylistProps) {
+  const {
+    playlist,
+    title,
+    onSavePlaylist,
+    enableOptions,
+    onSaveToSpotify,
+    onExportJSPF,
+  } = props;
+  const { spotifyAuth } = React.useContext(GlobalAppContext);
+  const showSpotifyExportButton = spotifyAuth?.permission?.includes(
+    "playlist-modify-public"
+  );
+  if (!playlist?.track?.length) {
+    if (playlist !== undefined) {
+      return <div id="title">No playlist was generated.</div>;
+    }
+    return null;
+  }
+  return (
+    <div>
+      <div id="playlist-title">
+        {enableOptions && (
+          <span className="dropdown pull-right">
+            <button
+              className="btn btn-info dropdown-toggle"
+              type="button"
+              id="options-dropdown"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="true"
+            >
+              <FontAwesomeIcon icon={faCog as IconProp} title="Options" />
+              &nbsp;Options
+            </button>
+            <ul
+              className="dropdown-menu dropdown-menu-right"
+              aria-labelledby="options-dropdown"
+            >
+              <li>
+                <a onClick={onSavePlaylist} role="button" href="#">
+                  Save
+                </a>
+              </li>
+              {showSpotifyExportButton && (
+                <>
+                  <li role="separator" className="divider" />
+                  <li>
+                    <a
+                      onClick={onSaveToSpotify}
+                      id="exportPlaylistToSpotify"
+                      role="button"
+                      href="#"
+                    >
+                      <FontAwesomeIcon icon={faSpotify as IconProp} /> Export to
+                      Spotify
+                    </a>
+                  </li>
+                </>
+              )}
+              <li role="separator" className="divider" />
+              <li>
+                <a
+                  onClick={onExportJSPF}
+                  id="exportPlaylistToJSPF"
+                  role="button"
+                  href="#"
+                >
+                  <FontAwesomeIcon icon={faFileExport as IconProp} /> Export as
+                  JSPF
+                </a>
+              </li>
+            </ul>
+          </span>
+        )}
+        <div id="title">{title}</div>
+      </div>
+      <div>
+        {playlist.track.map((track: JSPFTrack, index) => {
+          return (
+            <PlaylistItemCard
+              key={`${track.id}-${index.toString()}`}
+              canEdit={false}
+              track={track}
+              currentFeedback={0}
+              updateFeedbackCallback={() => {}}
```

Comment:
Once  #2534 is finished and merged, these two properties will need to be removed